js/notify/electron-notify.js

```diff
@@ -421,7 +421,7 @@ function setNotificationContents(notfWindow, notfObj) {
 
     // Display time per notification basis.
     let displayTime = notfObj.displayTime ? notfObj.displayTime : config.displayTime;
-
+    notfWindow.setAlwaysOnTop(true);
```

Comment:
this should be selective and should only be enabled if this user has "Always on Top" enabled on our app. If not, it'll be always on top even if our app is not in focus.

Reply:
@VishwasShashidhar ok, I fixed it.
Thanks